frontend-app/src/main/kotlin/no/nav/modialogin/persistence/jdbc/PostgresPersistencePubSub.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.modialogin.persistence.jdbc
+
+import kotlinx.coroutines.*
+import no.nav.modialogin.Logging.log
+import no.nav.modialogin.persistence.PersistencePubSub
+import org.postgresql.jdbc.PgConnection
+import org.postgresql.util.PSQLException
+import java.sql.SQLException
+import java.sql.SQLTransientConnectionException
+import javax.sql.DataSource
+
+class PostgresPersistencePubSub(
+    channelName: String,
+    private val dataSource: DataSource,
+) : PersistencePubSub(channelName, "postgres") {
+    override suspend fun subscribe(retryInterval: Long) {
+        while (running) {
+            try {
+                val listener = dataSource.connection.unwrap(PgConnection::class.java)
+                val stmt = listener.prepareStatement("LISTEN  persistence_updates")
+                stmt.execute()
+                while (true) {
+                    val notifications = listener!!.getNotifications(retryInterval.toInt()) ?: continue
+                    for (notification in notifications) {
+                        runBlocking(Dispatchers.IO) {
```

Comment:
Litt usikker på hvordan `runBlocking` til `Dispatchers.IO` fungerer under-the-hood.

Men litt avhengig av hvilken `Dispatcher` man er på når man kaller `subscribe` så tror jeg man risikerer at det fører til at man må bytte `Thread` i dette tilfellet. 

Kan isåfall flytte `runBlocking` ut av for-løkken, slik at man ikke må hoppe frem-og-tilbake mellom tråder.